### PR TITLE
1.8 crafting recipes

### PR DESCRIPTION
**Pull request by dequis**
Doing this manually since the old method to extract recipes required craftbukkit and craftbukkit is kill.

This isn't particularly exciting.

~~Opening PR to better keep track of what needs to be~~ **done**!. List mostly from #314 
- [x] Raw and Polished Diorite, Andesite and Granite
  - [x] Diorite
  - [x] Polished Diorite
  - [x] Andesite
  - [x] Polished Andesite
  - [x] Granite
  - [x] Polished Granite
- [x] Coarse Dirt
- [x] Red Sandstone, its variants, and slabs/stairs
  - [x] Red sandstone
  - [x] Red sandstone slab
  - [x] Red sandstone stairs
  - [x] Chiseled red sandstone
  - [x] Smooth red sandstone
- [x] Prismarine
- [x] Prismarine Bricks
- [x] Dark Prismarine
- [x] Sea Lantern
- [x] Slime Block - both ways
- [x] Iron Trapdoor
- [x] Iron door gives 3 instead of 1
- [x] Armor Stand
- [x] Leather from Rabbit Hide
- [x] Cooked Rabbit
- [x] Rabbit Stew
- [x] Chiseled Stone Bricks
- [x] Cracked Stone Bricks
- [x] Mossy Stone Bricks
- [x] Mossy Cobblestone
- [x] All new fences
  - [x] Oak
  - [x] Spruce
  - [x] Birch
  - [x] Jungle
  - [x] Acacia
  - [x] Dark oak
- [x] All new gates
  - [x] Oak
  - [x] Spruce
  - [x] Birch
  - [x] Jungle
  - [x] Acacia
  - [x] Dark oak
  - [x] ~~Gamer~~
- [x] All new doors
  - [x] Oak
  - [x] Spruce
  - [x] Birch
  - [x] Jungle
  - [x] Acacia
  - [x] Dark oak

Not going to do, at least for now:
- Banners

### CODE REVIEW

**Comment by dequis** [comment]
Okay I'm done writing recipes.

I bet there's at least one typo in there.

Now this needs testing and I don't feel like doing it, particularly with the awful crafting UI click handler which rejects completely valid operations even for known recipes.

**Comment by gdude2002** [comment]
That's terrible, @dequis 

Passed: http://bamboo.gserv.me/browse/GLOW-SRV-PR429-93

**Comment by turt2live** [comment]
What happened to "please limit scope" :(

**Comment by dequis** [comment]
It's limited to a single file being edited, and no banners :D

Seriously though, it would be silly to do any less than this.

**Comment by dequis** [comment]
Also it's 500ish lines where reviewing consists in either "check for typos" or "put shit in crafting grid in minecraft and see if the transaction gets rejected".

**Comment by turt2live** [comment]
I honestly didn't even look at the code, just the description :p

sent from mobile

File: src/main/resources/builtin/recipes.yml [hunk 1]
```diff
@@ -519,6 +529,37 @@ shapeless:
...
+- result:
+    type: STONE
+    damage: 1   # granite
+  ingredients:
+  - type: STONE
+    damage: 3   # diorite
+  - type: QUARTZ
+- result:
+    type: STONE
+    damage: 5   # andresite
```
Comment (turt2live) [inline src/main/resources/builtin/recipes.yml]: `Andesite`

**Comment by turt2live** [comment]
@dequis All doors result in a stack of 3. Your list does not account for this.

**Comment by dequis** [comment]
Good catch.

**Comment by turt2live** [comment]
\o/

**Comment by turt2live** [comment]
Existing cases for recipes are broken. For example, adesite shouldn't make a button (adesite is stone with a damage value).

**Comment by dequis** [comment]
Done. Affected recipes:

> comparator, repeater, stone slab, stone pressure plate, stone button

**Comment by SpaceManiac** [comment]
Merged in cf9580d4406cb5677b06fb5bc833f10213e625d8.

**Pull request closed without merging**